x-pack/plugins/observability/public/pages/rule_details/config.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RuleType, Rule } from '@kbn/triggers-actions-ui-plugin/public';
+
+type Capabilities = Record<string, any>;
+
+export type InitialRule = Partial<Rule> &
+  Pick<Rule, 'params' | 'consumer' | 'schedule' | 'actions' | 'tags' | 'notifyWhen'>;
+
+export function hasAllPrivilege(rule: InitialRule, ruleType?: RuleType): boolean {
```

Comment:
@fkanout Do we plan to move these functions in a common place? Same function is defined here https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/rules/config.ts#L64. Same is true for `hasExecuteActionsCapability`. Ideally we should move all these under triggers_actions_ui, but I guess we leave this for a follow up PR, right?

Reply:
@mgiota, Absolutely!